src/style.css

```diff
@@ -1,3 +1,68 @@
+:root {
+  --blue_color: #00acdc;
```

Comment:
The variable name should show their purpose(--active-color or something similar in this case)